src/lang/checkers/Cashflow.ml

```diff
@@ -78,10 +78,20 @@ module ScillaCashflowChecker
   (*     Initial traversal: Set every tag to NoInfo      *)
   (*******************************************************)
   
-  (* Lift Ident (n, rep) to Ident (n, (NoInfo, rep)) *)
-  let add_noinfo_to_ident i =
+  let add_tag_to_ident i tag =
```

Comment:
This seems to be a pretty general function. What do you think of moving this to `Syntax.ml` so different analyzers could reuse this definition?

Reply:
Well spotted, but it actually belongs in CashflowRep. If we move it to Syntax.ml, we restrict the annotations to being a string of pairs, which is not the case if we make the function specific to CashflowRep.